src/rgw/setup/rgw.py

```diff
@@ -157,7 +157,7 @@ def start(conf: MappedConf, index: str):
         log_path = Rgw._get_log_dir_path(conf)
         log_file = os.path.join(log_path, f'{COMPONENT_NAME}_startup.log')
         config_file = Rgw._get_rgw_config_path(conf)
-        RgwStart.start_rgw(conf, config_file, log_file, index)
+        RgwStart.start_service(conf, config_file, log_file, index)
```

Comment:
https://github.com/Seagate/cortx-rgw-integration/pull/54/files#r820413668

class & method name getting addressed in PR above